raiden/raiden_event_handler.py

```diff
@@ -546,18 +547,28 @@ class RaidenMonitoringEventHandler(RaidenEventHandler):
     # pylint: disable=no-self-use
 
     def on_raiden_event(self, raiden: RaidenService, event: Event):
-        if type(event) == EventPaymentReceivedSuccess:
-            self.handle_received_payment(raiden, event)
+        if type(event) == EventNewBalanceProofReceived:
+            self.handle_received_balance_proof(raiden, event)
 
         # Call base class to trigger other side-effects
         super().on_raiden_event(raiden, event)
 
-    def handle_received_payment(
+    def handle_received_balance_proof(
             self,
             raiden: RaidenService,
-            payment_received: EventPaymentReceivedSuccess,
+            new_balance_proof_event: EventNewBalanceProofReceived,
     ):
         log.info(
-            'Received payment, sending balance proof to Monitoring Service',
```

Comment:
receiving a payment and a new balance proof are not the same thing, please don't replace one with other

Reply:
as you can probably see, this is the old version, and I changed it to balance proofs

Reply:
another note for context: the above lines are from `RaidenMonitoringEventHandler` which was previously only a stub, (incorrectly) assuming to operate on `EventPaymentReceivedSuccess`.